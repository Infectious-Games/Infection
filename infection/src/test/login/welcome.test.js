import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Welcome from '../../views/login/welcome';;

test('should render Welcome', () => {
  const welcome = renderer
    .create(<Welcome></Welcome>)
    .toJSON();
  expect(welcome).toMatchSnapshot();
});