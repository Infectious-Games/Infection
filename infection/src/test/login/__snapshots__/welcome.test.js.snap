// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Welcome 1`] = `
<div
  className="welcome container"
>
  <div
    className="welcome row"
  >
    <div
      className="col-md-2"
    />
    <div
      className="welcome col-md-8"
    >
      <img
        alt="logo"
        className="img-responsive"
        src="Infection.jpg"
      />
    </div>
    <div
      className="col-md-2"
    />
  </div>
  <div
    className="row"
  >
    <div
      className="col-md-2"
    />
    <div
      className="col-md-8"
    >
      <h1
        className="welcome"
      >
        Welcome to the team!
      </h1>
      <p>
        Our mission is to stop infectious outbreaks threatening the United States
      </p>
    </div>
    <div
      className="col-md-2"
    />
  </div>
  <div
    className="login row"
  >
    <div
      className="col-md-4"
    />
    <div
      className="col-md-4"
    >
      <form
        className="login form-inline"
      >
        <div
          className="form-group"
        >
          <label
            className="control-label"
            htmlFor="formInlineName"
          />
           
          <input
            className="form-control"
            id="formInlineName"
            onChange={[Function]}
            onSubmit={[Function]}
            placeholder="Your Name"
            type="text"
            value=""
          />
        </div>
         
        <button
          className="btn btn-danger"
          disabled={false}
          onClick={[Function]}
          type="submit"
        >
          Enter Game
        </button>
      </form>
    </div>
    <div
      className="col-md-4"
    />
  </div>
</div>
`;
